Cannot find module 'xo' or its corresponding type declarations.

Hi,

When used with typescript (v3 and v4), I got following error:

```
node_modules/read-git-user/index.d.ts:1:23 - error TS2307: Cannot find module 'xo' or its corresponding type declarations.

1 import {Options} from 'xo';
                        ~~~~
```

Also it seems, `xo` is not used here. Removing this line corrects the error.

Thanks,

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,5 +1,3 @@
-import {Options} from 'xo';
-
 declare namespace readGitUser {
 	interface Options {
 		/**

